build-system/tasks/karma.conf.js

```diff
@@ -143,50 +149,77 @@ module.exports = {
       base: 'SauceLabs',
       browserName: 'chrome',
       version: 'latest',
+      maxDuration: SAUCE_MAX_DURATION_SECS,
+      commandTimeout: SAUCE_COMMAND_TIMEOUT_SECS,
+      idleTimeout: SAUCE_IDLE_TIMEOUT_SECS,
```

Comment:
You can define shared properties once and then use `Object.assign` to copy them across each SL config.

```js
const SAUCE_CONFIG_TIMEOUT = {
  maxDuration: SAUCE_MAX_DURATION_SECS,
  commandTimeout: SAUCE_COMMAND_TIMEOUT_SECS,
  idleTimeout: SAUCE_IDLE_TIMEOUT_SECS,
};

...

SL_Chrome_latest: Object.assign({
  base: 'SauceLabs',
  browserName: 'chrome',
  version: 'latest',
}, SAUCE_TIMEOUT_CONFIG),
```

Reply:
Cool! Done.